pkg/provider/resource_beacon_analytics.go

```diff
@@ -0,0 +1,306 @@
+					},
+				},
+			},
+		},
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				MarkdownDescription: "Resource ID of the cluster.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"cluster_id": schema.StringAttribute{
+				MarkdownDescription: "Cluster ID.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"connection_uri": schema.StringAttribute{
+				MarkdownDescription: "Cluster connection URI.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{plan_modifier.CustomConnection()},
+			},
+			"cluster_name": schema.StringAttribute{
+				MarkdownDescription: "Name of the cluster.",
+				Required:            true,
+			},
+			"phase": schema.StringAttribute{
+				MarkdownDescription: "Current phase of the cluster.",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					plan_modifier.CustomPhaseForUnknown(),
+				},
+			},
+			"project_id": schema.StringAttribute{
+				MarkdownDescription: "BigAnimal Project ID.",
+				Required:            true,
+				Validators: []validator.String{
+					ProjectIdValidator(),
+				},
+			},
+			"logs_url": schema.StringAttribute{
+				MarkdownDescription: "The URL to find the logs of this cluster.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"backup_retention_period": schema.StringAttribute{
+				MarkdownDescription: "Backup retention period. For example, \"7d\", \"2w\", or \"3m\".",
+				Optional:            true,
+				Computed:            true,
+				Validators: []validator.String{
+					BackupRetentionPeriodValidator(),
+				},
+				PlanModifiers: []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"cloud_provider": schema.StringAttribute{
+				Description: "Cloud provider. For example, \"aws\", \"azure\", \"gcp\" or \"bah:aws\", \"bah:gcp\".",
+				Required:    true,
+			},
+			"pg_type": schema.StringAttribute{
+				MarkdownDescription: "Postgres type. For example, \"epas\", \"pgextended\", or \"postgres\".",
+				Required:            true,
+				Validators: []validator.String{
+					stringvalidator.OneOf("epas", "pgextended", "postgres"),
+				},
+			},
+			"first_recoverability_point_at": schema.StringAttribute{
+				MarkdownDescription: "Earliest backup recover time.",
+				Computed:            true,
+				PlanModifiers:       []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+			},
+			"pg_version": schema.StringAttribute{
+				MarkdownDescription: "Postgres version. See [Supported Postgres types and versions](https://www.enterprisedb.com/docs/biganimal/latest/overview/05_database_version_policy/#supported-postgres-types-and-versions) for supported Postgres types and versions.",
```

Comment:
not all versions are supported for AC, need to remove the link to check "Supported Postgres types and versions", as the details in the specified link are DB cluster specific